Sources/NIOCore/AsyncSequences/NIOAsyncSequenceProducer.swift

```diff
@@ -83,7 +88,13 @@ public struct NIOAsyncSequenceProducer<
     Strategy: NIOAsyncSequenceProducerBackPressureStrategy,
     Delegate: NIOAsyncSequenceProducerDelegate
 >: Sendable {
-    /// Simple struct for the return type of ``NIOAsyncSequenceProducer/makeSourceAndSequence(backPressureStrategy:delegate:)``.
+    /// Simple struct for the return type of ``NIOAsyncSequenceProducer/makeSequence(of:backPressureStrategy:delegate:)``.
+    ///
+    /// This struct contains two properties:
+    /// 1. The ``NIOAsyncSequenceProducer/NewSequence/source`` which should be retained by the producer and is used
+    /// to yield new elements to the sequence.
+    /// 2. The ``NIOAsyncSequenceProducer/NewSequence/sequence`` which is the actual ``Swift/AsyncSequence`` and
+    /// should be passed to the consumer.
```

Comment:
FYI: no need to fully qualify the names, you could just ``sequence`` and it'll resolve properly :)

Thanks for the docs here!